prism/src/main/java/org/apache/falcon/resource/proxy/ExtensionManagerProxy.java

```diff
@@ -460,15 +473,9 @@ protected APIResult doExecute(String colo) throws FalconException {
                 }else{
                     processNames.add(entity.getName());
                 }
+                metaStore.storeExtensionJob(jobName, extensionName, feedNames, processNames, configBytes);
```

Comment:
why store is required in two places. its already stored before submitting entities